misc/ls-extensions/modules/partial-parser/build.gradle

```diff
@@ -28,7 +28,9 @@ dependencies {
     compileOnly project(':ballerina-parser')
     compileOnly project(':ballerina-lang')
     compileOnly project(':diagram-util')
+    compileOnly project(':formatter:formatter-core')
     compileOnly project(':language-server:language-server-commons')
+    compileOnly project(':language-server:language-server-core')
```

Comment:
It's not recommended to use `language-server-core` as a direct dependency. Shall we remove it?